### ISSUE TITLE
Ensure that all Shields text/background have sufficient contrast

### ISSUE THREAD

**Issue by tomlowenthal** [brave/brave-browser#1956]
Can check contrast performance with [this tool](http://accessible-colors.com). Ideally, we should be going for AAA, but AA should be okay.

**Comment by karenkliu** [comment]
@tomlowenthal @cezaraugusto 
Shields color update:
![image](https://user-images.githubusercontent.com/2388823/48646934-5c80cf80-e9b8-11e8-9030-6010170953f5.png)
Abstract link (for CSS): https://share.goabstract.com/4a733bfa-bf8e-45f2-89ac-e1b4c3908e3e

**Comment by karenkliu** [comment]
Closing this issue because comprehensive design specs and other issues are covered by #2565
